config/compiler-centos7-gcc62.yaml

```diff
@@ -2,10 +2,10 @@ compilers:
 - compiler:
     environment:
       set:
-        LD_LIBRARY_PATH: /cvmfs/sft.cern.ch/lcg/contrib/gcc/6.2.0binutils/x86_64-centos7/lib64::/cvmfs/sft.cern.ch/lcg/contrib/binutils/2.28/x86_64-centos7/lib:EXTRA_LIBS
+        LD_LIBRARY_PATH: /cvmfs/sft.cern.ch/lcg/contrib/gcc/6.2.0binutils/x86_64-centos7/lib64:/cvmfs/sft.cern.ch/lcg/contrib/binutils/2.28/x86_64-centos7/lib:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.28-19981/x86_64-slc6-gcc62-opt/lib:EXTRA_LIBS
     extra_rpaths:
-    - /cvmfs/sft.cern.ch/lcg/contrib/binutils/2.28/x86_64-centos7/lib
-    - /cvmfs/sft.cern.ch/lcg/contrib/binutils/2.28/x86_64-centos7/bin
+    - /cvmfs/sft.cern.ch/lcg/releases/binutils/2.28-19981/x86_64-slc6-gcc62-opt/bin
+    - /cvmfs/sft.cern.ch/lcg/releases/binutils/2.28-19981/x86_64-slc6-gcc62-opt/lib
```

Comment:
The configuration file is for a `centos7` platform while this link points to a compiler built for `slc6`.